ui/webui/src/apis/storage.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from "cockpit";
+
+const storageClient = address => {
+    return cockpit.dbus(
+        "org.fedoraproject.Anaconda.Modules.Storage",
+        { superuser: "try", bus: "none", address }
+    );
+};
+
+export const applyPartitioning = (address, partitioning) => {
```

Comment:
@VladimirSlavik @M4rtinK do you think you want to have some kind of doc strings for these?

Reply:
Yes, please! :)

Reply:
Added

Reply:
The localization APIs are still in the react code. I will first merge https://github.com/rhinstaller/anaconda/pull/3924 and then move also these APIs to apis folder and document them.